PowerEditor/src/tools/NppModernShell/README.md

```diff
@@ -0,0 +1,61 @@
+# Windows 11 Modern UI Context Menu integration
+
+The purpose of this project is to allow Notepad++ to integrate into the new Windows 11 right-click context menu.
+Doing this requires two new things.
+
+* A dll library with some COM objects that the shell can communicate with.
+* A Sparse Package containing the metadata for the COM server.
+
+To build this, the following steps needs to be taken:
+
```

Comment:
I would add this line:
0. Install Windows 11 SDK from: https://developer.microsoft.com/en-us/windows/downloads/windows-sdk/

:)